manifest_staging/chart/gatekeeper-operator/templates/gatekeeper.yaml

```diff
@@ -458,7 +458,7 @@ spec:
   template:
     metadata:
       annotations: 
-{{- toYaml .Values.podAnnotations | trim | nindent 8 }}
```

Comment:
why are we removing trim?

Reply:
No clue how that got in there, I can revert

Reply:
Though this is likely the result of someone modifying the chart but not the generation pipeline

Reply:
Yep, missing change in pipeline, added